modules/Bio/EnsEMBL/Production/Pipeline/EarlyDumps/MetadataReport.pm

```diff
@@ -106,11 +106,15 @@ sub species_wise_metadata_files {
 	my ( $self, $dump_path) = @_;
 
 	my @division_list = @{$self->param('division_list')};
+	
+	if(@{$self->param('division')}){
+	  @division_list = @{$self->param('division')};
+	}
 	my @non_vert_combine_files = ();
 	my $cmd_str = "";
 	my $division_specific_param = {    'ftp_dir'          => $self->param('ens_ftp_dir'),
-                                           'base_path'        => catdir($self->param('base_path'),  "/release-".$self->param('ens_version')), 
-                                           'dump_dir'         => catdir($self->param('base_path'),  "/release-".$self->param('ens_version')),
+                                           'base_path'        => catdir($self->param('base_path'),  "/release-".$self->param('ens_version')."/ftp_dumps"), 
```

Comment:
For these lines, may be we should just change the SOP, so the base_path is actually the base_path, and not forcing the subpath structure